### PR TITLE
Add missing migrations for tests app/models

### PR DESCRIPTION
**Pull request by sergei-maertens**
Simply checking out master and running makemigrations produces these missing migrations. Run on the lowest supported versions of Python and Django (3.8 and 3.2 respectively).

Separate PR for this to not distract from the main PR, for which a new model field is added to be able to test the behaviour for #698.

### CODE REVIEW

**Comment by last-partizan** [comment]
@sergei-maertens i thin, for the test app it's better to just remove initial migration and recreate it.

And don't forget to run `black modeltranslation *.py` on your changes.

**Comment by sergei-maertens** [comment]
> @sergei-maertens i think, for the test app it's better to just remove initial migration and recreate it.
> 
> And don't forget to run `black modeltranslation *.py` on your changes.

Sure, works for me! I'll update the PR

w/r to black - is it only applied on the app itself? I noticed that the tests themselves do not have consistent code formatting, that's why I didn't run it in the first place as it was unclear that black is used. The contributor docs only make mention of flake8.

**Comment by sergei-maertens** [comment]
Rebased and updated with re-generated initial migration.

File: modeltranslation/tests/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+import django.db.models.deletion
 import modeltranslation.tests.models
 
 
 class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('auth', '0001_initial'),
+        ('auth', '0013_group_name_de_group_name_en'),
```
Comment (last-partizan) [inline modeltranslation/tests/migrations/0001_initial.py:14]:
```suggestion
        ('auth', '0001_initial'),
```

**Comment by last-partizan** [comment]
> w/r to black - is it only applied on the app itself? I noticed that the tests themselves do not have consistent code formatting

It is applied, but for whatever reason we have `skip-string-normalization = true` in pyproject.toml. I'll fix it after we merge your changes.

File: modeltranslation/tests/migrations/0001_initial.py (hunk 1 above)
Comment (sergei-maertens) [reply modeltranslation/tests/migrations/0001_initial.py]: Updated!

**Pull request merged**